code/world.dm

```diff
@@ -397,11 +397,6 @@ var/f_color_selector_handler/F_Color_Selector
 	tick_lag = MIN_TICKLAG//0.4//0.25
 //	loop_checks = 0
 
-	// Load in the current commit SHA from TGS if avail, by MCterra10
-	if(TgsAvailable())
-		var/datum/tgs_revision_information/rev = TgsRevision()
-		vcs_revision = rev.commit
```

Comment:
we still need to set this tho.. either get rid of the variable entirely or set the define to it

Reply:
Variable is set by a tgs event script before compilation, similar to how Jenkins did it.

Reply:
Can't you only do that with definitions?

Reply:
i see. you are doing it by definition

Reply:
Pre compilation event script in TGS has the following code:

```bash
var/global/vcs_revision = "$(git rev-parse HEAD)"
var/global/vcs_author = "$(git log -1 --pretty=format:'%an')"
```

This is stamped to the `_std/__build.dm` file. This is almost exactly how Jenkins did it.